### DISCUSSION TITLE
IMAGE UPLOAD

### DISCUSSION THREAD

**Discussion by How-do-i-get-your-attention** [community/community#52393] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

i feel so sad.. i have a mission on github...
i try make a story in github.. code ^^

and then i want to add my profile image wtf guys .... im a developer need to be open souce  ^^ SVG GUY..

COME ON NOW I NEED TO MAKE a PNG I will wait for you answe plz no png or jpg.. svg yaaa love it ..

![PHOTO-BUG-HAHAHA](https://user-images.githubusercontent.com/129417986/230792677-734b938b-41b3-435d-a8c8-23f76005175a.png)

**Comment by How-do-i-get-your-attention** [comment]
BTW I PUBLIC THIS IN MY STORY IN GITHUB
https://github.com/How-do-i-get-your-attention/Developer/commit/43098059b9d61f6147e2ec99367daad8e44580db  :////

**Comment by SilasRodrigues19** [comment]
Hey @How-do-i-get-your-attention 

You can convert your SVG to PNG, I usually use this site here:

https://convertio.co/pt/svg-png/

**Comment by How-do-i-get-your-attention** [comment]
damm it  :/ oka then
